interface-definitions/include/accel-ppp/client-ip-pool.xml.i

```diff
@@ -0,0 +1,46 @@
+<!-- include start from accel-ppp/client-ip-pool.xml.i -->
+<tagNode name="client-ip-pool">
+  <properties>
+    <help>Client IP Pool</help>
```

Comment:
Nitpick: why capitalize the word "pool"?

Reply:
done